tests/unit/amo/pages/TestHome.js

```diff
@@ -649,14 +649,14 @@ describe(__filename, () => {
       expect(heroRecommendation).toHaveProp('errorHandler', errorHandler);
       expect(heroRecommendation).toHaveProp(
         'shelfData',
-        createInternalHeroShelves(heroShelves).primary,
+        createInternalHeroShelves(heroShelves, 'en-US').primary,
```

Comment:
Should we have a `createInternalHeroShelvesWithDefaultLang` helper?

Reply:
Yes, I'll add one.